src/components/SRNEntry.vue

```diff
@@ -59,15 +62,10 @@ export default {
     return {
       userIDList: [{ userID: "" }],
       invalidInputMessage: null,
-      validated: true,
-      maxLength: 10,
-      /*this count is used to check how many times the user has been validated. 
-      For now, the count has three possible states : -1 -> the initial state, if the user is validated in the first try. This will set the flag to valid
-      0 -> user has been authenticated once, and is being authenticated again. If yes, then the flag is set to valid. 
-      Otherwise, the state changes to 1, this means the user has tried getting authenticated twice, but is invalid. 
-
-      */
-      validateCount: -1,
+      doesUserExist: false,
```

Comment:
Add one comment like this
```js
doesUserExist: false, // whether the user exists in the backend database
```

Reply:
What I do personally, before I make the PR, I add one liner comments to everything, methods, computed variables, data properties. Helps me when I revisit the code after months